client/my-sites/plan-features/test/header.jsx

```diff
@@ -0,0 +1,147 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+/**
+ * Internal dependencies
+ */
+import { BareComponent as PlanFeaturesHeader } from '../header';
+import {
+	PLAN_FREE,
+	PLAN_PREMIUM
+} from 'lib/plans/constants';
+
+describe( 'plan-features-header', () => {
+	it( 'should have component class', () => {
+		const header = shallow(
+			<PlanFeaturesHeader
+				billingTimeFrame={ 'per month' }
+				current={ false }
+				planType={ PLAN_PREMIUM }
+				popular={ true }
+				rawPrice={ 8.25 }
+				currencyCode={ 'USD' }
+				title={ 'Premium' }
+			/>
+		);
+		expect( header.hasClass( 'plan-features__header' ) ).to.equal( true );
+	} );
+
+	describe( 'prices display correctly', () => {
+		it( 'free', () => {
+			const header = shallow(
+				<PlanFeaturesHeader
+					billingTimeFrame={ 'for life' }
+					current={ false }
+					planType={ PLAN_FREE }
+					popular={ true }
+					rawPrice={ 0 }
+					currencyCode={ 'USD' }
+					title={ 'Premium' }
+				/>
+			);
+			expect( header.find( '.plan-features__header-price' ).render().children().html() ).to.equal(
+				'<sup class="plan-features__header-currency-symbol">$</sup>0'
+			);
```

Comment:
These tests appear to be actually testing some component that builds a formatted currency string, but in actuality are testing the `<PlanFeaturesHeader />` Maybe it would be more useful if we had a `<PriceDisplay { ...{ value, units } } />` component and could test it in isolation.